tasks/create_vpc_subnet.yml

```diff
@@ -34,8 +34,8 @@
     # Looked-up values
     vpc_id: "{{ aws_vpc_subnet_vpc_id }}"
     cidr: "{{ subnet.cidr }}"
-    # Optional values
-    az: "{{ subnet.region | default(aws_vpc_subnet_default_region) }}{{ subnet.az | default(omit) }}"
+    # Optional values    
+    az: "{{ subnet.az | default(omit) }}"
```

Comment:
Really good catch. :+1: This is also the reason to consume our own roles for CI tests of other AWS stuff.